schemas/error.yml

```diff
@@ -40,7 +40,6 @@
     - name: stack_trace
       level: extended
       type: wildcard
-      index: false
```

Comment:
https://webmat.keybase.pub/funny-stuff/dalai_lasers.gif

Reply:
Yeah, I removed it since it's unnecessary with `type: wildcard`.

However, as I think about this case further, removing `index: false` will mean `error.stack_trace` will be indexed even for `keyword`-fallback cases like the ES 6.x template or `--oss`. 🤔

Reply:
If we did want to leave `index: false` intact for any non-wildcard use cases, two considerations:

1. How to best capture this in the field definition. Having `type: wildcard` and `index: false` could create misunderstanding and confusion.
 
2. Address that the `index: false` and `doc_values: false` settings are being stripped in the ES 6 template (what originally led me to remove this line).

Reply:
It's going to feel a bit dirty, but I wonder if we shouldn't hardcode the details for this fallback, or something.

Doing like `event.original` and leaving it non-indexed in the main schema is a no-go, as `wildcard` on stack traces will be objectively fantastic, for developers.

On the other hand, redoing how basic => oss translation works to better handle this edge case is way out of scope, IMO. This is just a "best effort" equivalency. From ECS 1.8 moving forward, ECS is using basic data types. (<= this "period" should be said out loud). So folks are free to override what's happening with this field however works best for them, if they need to stick to OSS and want to use this field. But really they should upgrade to the free basic option, which supports `wildcard`.

So in short, I think we should:

- leave the main schema using `wildcard` and no `index` attribute
- in the OSS fallback we should hardcode something so that `error.stack_trace` puts this back to `keyword` and `doc_values:false` and leave it at that.

I think this field and event.original were the only two, correct?

Reply:
> I think this field and event.original were the only two, correct?

Correct.

I agree with the best-effort support around these nuances and favoring the simple workaround for now :-)

Reply:
A "quick and simple" implementation to address proposed in #1176